modules/nf-core/gatk4/markduplicatesspark/environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::gatk4=4.4.0.0
+  - conda-forge::openjdk=8.0.312
```

Comment:
is this really equivalent to `container "nf-core/gatk:4.4.0.0"`?
Because for me that's the mulled container, the one here on quay.io/nf-core is the one from the broad...